s3-source-connector/src/main/java/io/aiven/kafka/connect/s3/source/config/S3SourceConfig.java

```diff
@@ -179,6 +173,11 @@ private static void addOtherConfig(final S3SourceConfigDef configDef) {
                 "Value converter", GROUP_OTHER, awsOtherGroupCounter++, // NOPMD
```

Comment:
It's a bad idea to define configurations with the same name as the framework. One will take precedence, and you can't be sure if the default you're specifying is correct or even present.